docs/designs/data-models.md

```diff
@@ -0,0 +1,116 @@
+# Data Models Dictionary
+
+This document presents the data model used in this software. It describes different domain entities, fields within
```

Comment:
consider incorporating a unique identifier for each entity in the data model

Reply:
Good point. I will do